src/documentation/app/Home.jsx

```diff
@@ -89,15 +89,15 @@ function Home() {
           here at Buffer
         </Header>
         <Places>
-          <Place href="/#/Guides/guides">
+          <Place href="/Guides/guides">
```

Comment:
not blocking but could this be `/guides` without repeating? and so on for the other two

Reply:
I agree with Juliana, if it's an easy change, could we drop one of them? 🙂